core/scss/components/card/card.scss

```diff
@@ -1,1073 +1,27 @@
-          }
-
-          button {
-            @include vr-spacing(p, 0);
-          }
-        }
-
-        i.icon-arrow-next:before {
-          display: inline-block;
-          transform: rotate(90deg);
-          transition: all 0.25s ease 0s;
-        }
-
-        &.open-panel i.icon-arrow-next:before {
-          transform: rotate(-90deg);
-        }
-      }
-
-      footer {
-        width: 100%;
-
-        @include vr-spacing(ph, 1.5);
-        @include vr-spacing(pb, 1);
-
-        &.footer-menu {
-          display: flex;
-          color: $md-gray-40;
-          background-color: $md-gray-10;
-          justify-content: space-between;
-          align-items: center;
-
-          @include vr-spacing(pt, 0.5, 1px);
-          @include vr-spacing(pb, 0.5, 2px);
-          @include vr-spacing(ph, 1.5);
-
-          .footer-content {
-            display: flex;
-            width: 100%;
-            min-height: rem(20);
-            align-items: center;
-          }
-
-          &.overview-footer-content-trial {
-            .footer-content {
-              justify-content: space-between;
-            }
-          }
-
-          &.overview-footer-content {
-            .footer-content {
-              justify-content: flex-end;
-            }
-          }
-
-          .footer-icons {
-            display: flex;
-            font-size: rem(20);
-            align-items: center;
-
-            a {
-              color: $md-gray-40;
-
-              &:focus {
-                box-shadow: none;
-              }
-            }
-
-            .menu-icon {
-              &:hover {
-                cursor: pointer;
-              }
-            }
-          }
-
-          .right-footer-icons {
-            justify-content: flex-end;
-          }
-        }
-      }
-    }
-
-    &.medium,
-    &.md-card--medium,
-    &.large,
-    &.md-card--large,
-    &.full,
-    &.md-card--full {
-      > article {
-        header {
-          @include vr-spacing(ph, 1.5);
-          @include vr-spacing(pt, 1.5);
-        }
-      }
-    }
-
-    .full.no-padding {
-      > article {
-        header {
-          padding: 0;
-        }
-      }
-    }
-
-    .card-menu {
-      width: 0;
-      height: 0;
-      overflow: hidden;
-      transform: translateY(100%);
-      transition: transform 0.6s ease;
-
-      &.open {
-        position: absolute;
-        top: 0;
-        width: 100%;
-        height: 100%;
-        color: $md-gray-30;
-        background-color: rgba(52, 53, 55, 0.98);
-        transform: translateY(0);
-        transition: transform 0.6s ease;
-      }
-
-      .card-menu-heading {
-        @include vr-spacing(p, 1.5);
-
+    position: relative;
+    display: flex;
+    color: $text-color;
+    vertical-align: top;
+    background-color: $card-color;
+    border-radius: $card-border-radius;
+    box-shadow: 0px 1px 2px 1px rgba(52, 53, 55, 0.18);
+    flex-direction: column;
+      .md-card-section {
```

Comment:
What's the reason for card-section vs card-full-section? Seems like it could be consolidated with a prop/input.